gatherling/event.php

```diff
@@ -1167,7 +1167,7 @@ function monthDropMenu($month)
 
 function structDropMenu($field, $def)
 {
-    $names = ['Swiss (Blossom)', 'Swiss', 'Single Elimination', /*"Round Robin",*/ 'League'];
+    $names = ['Swiss (Blossom)', /*'Swiss',*/ 'Single Elimination', /*"Round Robin",*/ 'League'];
```

Comment:
This was done in #8.  You'll want to rebase your other changes ahead of that.

Reply:
Ok, gonna check that

Reply:
OK, updated